src/data/meta.ts

```diff
@@ -195,7 +195,7 @@ export class MetaDataManager {
     return this.getSensor({ id: 'jhu-csse', signal: 'confirmed_7dav_incidence_prop' });
```

Comment:
This is the JHU signal we are removing...  Doesnt that mean we can get rid of this method too?